web/src/bootstrap_typeahead.ts

```diff
@@ -195,7 +195,7 @@ const HEADER_ELEMENT_HTML =
     '<p class="typeahead-header"><span id="typeahead-header-text"></span></p>';
 const CONTAINER_HTML = '<div class="typeahead dropdown-menu"></div>';
 const MENU_HTML = '<ul class="typeahead-menu" data-simplebar></ul>';
-const ITEM_HTML = "<li><a></a></li>";
+const ITEM_HTML = '<li><a class="typeahead-item-link"></a></li>';
```

Comment:
Should we put a `typeahead-item` class on the `li` too? Seems like it might be a good idea for consistency.

Reply:
yes as it will standardize and simplify the styling or behavior control for the items

Reply:
Sure!